kernel/Dockerfile

```diff
@@ -18,7 +19,7 @@ RUN if [ $DEBUG -ne "0" ]; then \
     fi
 
 # Apply local patches
-COPY patches-4.9 /patches
+COPY patches-${KERNEL_SERIES} /patches
```

Comment:
There was an issue in the past if there were no patches, as the directory would be empty, so not in git.